flink-streaming-java/src/main/java/org/apache/flink/streaming/api/checkpoint/ListCheckpointed.java

```diff
@@ -41,6 +41,7 @@
 	 * @param checkpointId The ID of the checkpoint.
 	 * @param timestamp Timestamp of the checkpoint.
 	 * @return The operator state in a list of redistributable, atomic sub-states.
+	 *         Should not return null, but empty list instead.
```

Comment:
Oh, plus `null` is semantically indifferent from empy list in this case. So for the sake of consistency, I give this advice in the doc.